lib/mix/test/fixtures/umbrella_dep/deps/umbrella/apps/foo/lib/foo_struct.ex

```diff
@@ -0,0 +1,3 @@
+defmodule FooStruct do
+  defstruct [:bar]
+end
```

Comment:
Instead of adding a new module, let's just write to `lib/foo.ex` a file with a struct. We should prefer to generate the data in the test itself rather than rely on fixtures. :)